hack/olm.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# olm.sh - run/cleanup the operator with OLM
+#
+# USAGE
+#    olm.sh -a run/cleanup -c container_url:tag
+# OPTIONS
+#    -a=     Action                   run/cleanup the operator installation
```

Comment:
Not sure if we need an explicit option for the action. Can't we say that `$1` needs to always be an action and check for that?